### PR TITLE
Fixes #35437 - host details jobs change from list to table

### PR DESCRIPTION
**Pull request by MariaAga**
Its more correct to use table in this case and its easier for qe to test

### CODE REVIEW

File: webpack/react_app/components/RecentJobsCard/RecentJobsTable.js [hunk 1]
```diff
@@ -33,55 +25,50 @@ const RecentJobsTable = ({ status, hostId }) => {
...
-                    ]}
-                  />
-                </DataListItemRow>
-              </DataListItem>
-            )
-          )}
-      </SkeletonLoader>
-    </DataList>
+        <Tbody>
+          {jobs?.length &&
```
Comment (adamruzicka) [inline webpack/react_app/components/RecentJobsCard/RecentJobsTable.js:45]: If there are no jobs, the card just says `0`
image

Maybe `map`-ing over an empty array should be valid

**Review by adamruzicka — CHANGES_REQUESTED** [review]

**Comment by adamruzicka** [comment]
Also needs a rebase

**Comment by MariaAga** [comment]
rebased and fixed the 0 issue by moving `jobs?.length` up in the tree to trigger the empty state message

**Review by adamruzicka — APPROVED** [review]
ACK

**Comment by adamruzicka** [comment]
Thank you @MariaAga !

**Pull request merged**